third_party/terraform/website/docs/provider_reference.html.markdown

```diff
@@ -1,22 +1,16 @@
 ---
 layout: "google"
-page_title: "google provider reference"
+page_title: "Google Provider Configuration Reference"
 sidebar_current: "docs-google-provider-reference"
 description: |-
-  The Google provider is used to configure your GCP project, location, and creds
+  Configuration reference for the Google provider for Terraform.
 ---
 
 # Google Provider Configuration Reference
 
--> Want more information about upgrading from `1.X` of the provider to `2.0.0`? Check out the [2.0.0 Upgrade Guide](https://terraform.io/docs/providers/google/version_2_upgrade.html).
-
-The `google` and `google-beta` provider blocks are used to configure default values for
-your GCP project and location (`zone` and `region`), and add your credentials.
-
--> You can avoid using a provider block by using environment variables. Every
```

Comment:
Breaking down one of the examples;

> The default project to manage resources in. If another
> project is specified on a resource, it will take precedence. Alternatively, this
> can be specified using the specified using the `GOOGLE_PROJECT` environment
> variable, or any of the following ordered by precedence.

> The default project to manage resources in. If another project is specified on a resource, it will take precedence.

clearly specifies the relationship between provider <-> resource

> Alternatively, this can be specified using the specified using the `GOOGLE_PROJECT` environment variable, or any of the following ordered by precedence.

I think that the `Alternatively,` implies the relationship there- that explicit values take precendence -strongly enough, especially because it should be the default assumption of Terraform users.  Do you think it's insufficient?